shell/pages/auth/login.vue

```diff
@@ -64,7 +64,12 @@ export default {
   },
 
   computed: {
-    ...mapGetters({ t: 'i18n/t' }),
+    ...mapGetters(['isSingleProduct']),
+    ...mapGetters({ t: 'i18n/t', hasMultipleLocales: 'i18n/hasMultipleLocales' }),
```

Comment:
We don't want to show LocalSelector when harvester is standalone mode, right?

How about using `isStandaloneHarvester`?
```
mapGetters(['isStandaloneHarvester']),
```

Reply:
Fixed